libs/components/lookup/src/lib/modules/country-field/clone-country-data.ts

```diff
@@ -0,0 +1,26 @@
+import { type Country } from 'intl-tel-input/data';
+
+import type { SkyCountryFieldCountry } from './types/country';
+
+/**
+ * Clone the country data for each component instance to avoid overwriting
+ * the global country data.
+ */
+export function cloneCountryData(data: Country[]): SkyCountryFieldCountry[] {
```

Comment:
Created these map functions so that we can keep a better eye on any type changes from `intl-tel-input` (JSON.stringify/parse returns an `any` type).